src/imgix-core-js.js

```diff
@@ -1,38 +1,32 @@
 (function (global, factory) {
   if (typeof define === 'function' && define.amd) {
-    define('Imgix', ['exports', 'md5', 'js-base64', 'crc'], factory);
+    define('Imgix', ['exports', 'md5', 'js-base64'], factory);
   } else if (typeof exports !== 'undefined') {
-    module.exports = factory(exports, require('md5'), require('js-base64').Base64, require('crc'));
+    module.exports = factory(exports, require('md5'), require('js-base64').Base64);
   } else {
     var mod = {
       exports: {}
     };
-    global.ImgixClient = factory(mod.exports, global.md5, global.Base64, global.crc);
+    global.ImgixClient = factory(mod.exports, global.md5, global.Base64);
   }
-})(this, function (exports, _md5, _jsBase64, _crc) {
+})(this, function (exports, _md5, _jsBase64) {
   var md5 = _md5;
   var Base64 = _jsBase64.Base64 || _jsBase64;
-  var crc = _crc;
 
   var VERSION = '1.4.0';
-  var SHARD_STRATEGY_CRC = 'crc';
-  var SHARD_STRATEGY_CYCLE = 'cycle';
   var DOMAIN_REGEX = /^(?:[a-z\d\-_]{1,62}\.){0,125}(?:[a-z\d](?:\-(?=\-*[a-z\d])|[a-z]|\d){0,62}\.)[a-z\d]{1,63}$/i;
   var DEFAULTS = {
     host: null,
-    domain: null,
-    domains: [],
+    domain: '',
```

Comment:
Can we continue using `null` as the default? Seems like it would be easier to catch when the option is omitted if it defaults to `null`.

Reply:
Ah yea, good catch